celery_app/tasks.py

```diff
@@ -0,0 +1,24 @@
+from celery_app.celery import celery_app as celery
+from random import choice
+
+
+@celery.task
+def add(x, y):
+    return x + y
+#
```

Comment:
what is this add task being used for in the application?

Reply:
This was supposed to be removed as well.